scripts/wsrep_sst_clone.sh

```diff
@@ -323,7 +323,7 @@ setup_certificates_for_donor()
 
     # REQUIRE_SSL is used during clone user creation on Donor server
     REQUIRE_SSL=""
-    if [ -n "$server_ssl_ca" ] && [ -n "$server_ssl_cert" ] && [ -n "$server_ssl_key" ]; then
+    if [ -n "$server_ssl_ca" ] && [ -n "$server_ssl_cert" ] && [ -n "$server_ssl_key" ] && [ ! "$server_ssl_ca" == "NULL" ]; then
```

Comment:
In Bash, for string equality, you should use = rather than == when inside [ ... ] (the classic test command). == is supported by [[ ... ]] (the extended test syntax), but not guaranteed with [ ... ] in all POSIX shells.

Please change to `[ "$server_ssl_ca" != "NULL" ]`

Reply:
Forget it I logout login again and I was able to see the dropdown now... @kamil-holubicki